{
 "bad": [
  "ZEO.ClientStorage.ClientStorage",
  "_compat.quote",
  "baduri",
  "basef",
  "convert",
  "deltaf",
  "f",
  "klass",
  "mock",
  "mock.patch",
  "os.path.exists",
  "os.path.join",
  "os.path.normpath",
  "resolver",
  "resolver.interpret_kwargs",
  "sm",
  "urllib.quote",
  "urlparse.parse_qsl",
  "urlparse.urlsplit",
  "zodburi.__init__._get_dbkw",
  "zodburi.__init__._parse_bytes",
  "zodburi.__init__._resolve_uri",
  "zodburi.__init__.bytes_parameters",
  "zodburi.__init__.connection_parameters",
  "zodburi.__init__.has_units",
  "zodburi.__init__.parameters",
  "zodburi.__init__.units"
 ],
 "deps": {
  "ZConfig": [
   "zconfig/conda-forge/noarch/zconfig-3.5.0-py_0"
  ],
  "ZEO": [],
  "ZODB": [
   "zodb/conda-forge/noarch/zodb-5.4.0-py_1"
  ],
  "_compat": [],
  "baduri": [],
  "basef": [],
  "convert": [],
  "deltaf": [],
  "f": [],
  "factory": [
   "factory_boy/conda-forge/noarch/factory_boy-2.11.1-pyh9f0ad1d_1"
  ],
  "io": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "klass": [],
  "mock": [],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "pkg_resources": [
   "setuptools/conda-forge/linux-64/setuptools-41.2.0-py27_0"
  ],
  "re": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "resolver": [],
  "shutil": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "sm": [],
  "tempfile": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "unittest": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "urllib": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "urlparse": [],
  "zodburi": []
 }
}